ifrs17/DataModel/DataStructure.ipynb

```diff
@@ -2097,16 +2092,46 @@
                 "\n    [Dimension(typeof(Scenario))]",
                 "\n    public string Scenario { get; init; }",
                 "\n",
+                "\n    public Args(int year, int month, string scenario)",
+                "\n    {",
+                "\n        Year = year;",
+                "\n        Month = month;",
+                "\n        Scenario = scenario;",
+                "\n    }",
+                "\n}"
+            ],
+            "metadata": {},
+            "execution_count": 0,
+            "outputs": []
+        },
+        {
+            "cell_type": "markdown",
+            "source": [
+                "Partitioned Args are used to hold data related to the partition and additional information required to performe calculation at import and report. ",
```

Comment:
Update this text. This one is a repetition of the above. 
Here you should described PartitionedArgs.